src/Tools/ResponseStrategies/ResponseFileStrategy.php

```diff
@@ -25,15 +33,20 @@ public function __invoke(Route $route, array $tags, array $routeProps)
     protected function getFileResponse(array $tags)
```

Comment:
Can we pluralize these methods now?

```suggestion
     protected function getFileResponses(array $tags)
```